src/styles/components/_subscribe.scss

```diff
@@ -0,0 +1,86 @@
+@import '../settings/colors';
+@import '../tools/breakpoints';
+
+.w-subscribe {
```

Comment:
Might need to enforce some dimensions on the heading so it doesn't touch the edges on smaller screen sizes. At around 850px it touches the edges of the screen.

Reply:
Also I think we're not quite matching the design of Jiwoong's mocks.
https://gallery.googleplex.com/projects/MCHbtQVoQ2HCZciehgH2S_Ws/files/MCGT2l9AragxTMzZaGwrrRpy

Reply:
I don't think I had seen this design before, will update to look like this then.

Reply:
oh sorry. I think it's linked from the slides deck where we were discussing the design.

Reply:
the things I was looking for us to match were around the styling of the input elements (padding, border color/contrast). The checkbox styles and their spacing.